### PR TITLE
Issue#1 

### PR DESCRIPTION
**Pull request by Dacalez**
Created test for api, reducer, action and MessageList component

### CODE REVIEW

File: 00 Base/src/API/myApi.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { setMessage, getListOfMessages } from './myApi';
```
Comment (dailymp) [inline 00 Base/src/API/myApi.test.js]: You should use typescript here too

File: 00 Base/src/components/MessageList/MessageList.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
```
Comment (dailymp) [inline 00 Base/src/components/MessageList/MessageList.test.js]: Convert this to typescript

File: 00 Base/src/redux/actions/MessagesActions.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            let result = {
+                type: actions.GET_ALL,
+                payload: response,
+            }
+
+            expect(actions.getAllMessages()).toBe(result);
+        })
+    })
+
+    it('should get all messages returns all messages', () => {
```
Comment (dailymp) [inline 00 Base/src/redux/actions/MessagesActions.test.js]: This description not correspond

File: 00 Base/src/redux/actions/MessagesActions.test.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+    // it('should getAllMessages to call getListOfMessages', () =>{
+    //     actions.getAllMessages();
+    //     expect(getListOfMessages).toHaveBeenCalled();
+    // })
+
+    // it('should get all messages to call getListOfMessages', () =>{
+    //     actions.addMessage(newMessage);
+    //     expect(setMessage).toHaveBeenCalledWith(newMessage);
+    // })
```
Comment (dailymp) [inline 00 Base/src/redux/actions/MessagesActions.test.js]: Why this commented code! Please fix it

File: 00 Base/src/redux/reducers/Mesages.Reducer.test.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        expect(MessagesReducer({}, getAllAction)).toEqual({messages: getAllAction.payload});
+    })
+    
+    it('should add a new message with ADD_MESSAGE data', () =>{
+        const messages = getAllAction.payload;
+        expect(MessagesReducer({messages: messages}, addMessageAction ).messages).toHaveLength(2);
+    })
+})
+
+
```
Comment (dailymp) [inline 00 Base/src/redux/reducers/Mesages.Reducer.test.js]: Did you check the total covarege? We need at least 75%

**Review by dailymp — CHANGES_REQUESTED** [review]
Please try to add more tests, and check the coverage! We need at least 75%

File: 00 Base/src/redux/reducers/Mesages.Reducer.test.js (hunk 5 above)
Comment (Dacalez) [reply 00 Base/src/redux/reducers/Mesages.Reducer.test.js]: I checked coverage per file, but I thought 75% total coverage was only for the third task. I will increase it.

Comment (dailymp) [reply 00 Base/src/redux/reducers/Mesages.Reducer.test.js]: Ok, thats nice

**Comment by dailymp** [comment]
@Dacalez The PR sould be from your branch to develop branch is the correct way of working

**Comment by dailymp** [comment]
> Created test for api, reducer, action and MessageList component

The PR sould be from your branch to develop branch is the correct way of working

**Comment by Dacalez** [comment]
I have upload the changes, added more test and changed the PR base as well

**Review by dailymp — CHANGES_REQUESTED** [review]
You project is not compiling well, is missing a loader
image

**Comment by dailymp** [comment]
> I have upload the changes, added more test and changed the PR base as well

Thanks for your changes, it seems a better work now, but still pending something, because is missing a loader

**Review by dailymp — CHANGES_REQUESTED** [review]
Please can you increase this specific coverage on MessageActions.ts? Te coverage here is less than 50%
image
This is failing 
image
And this is failing too:
image

**Comment by Dacalez** [comment]
I have upload the fixes. The coverage of MessageActions.ts has been increased, the problems in tests and with the style module have been fixed

**Comment by dailymp** [comment]
> I have upload the fixes. The coverage of MessageActions.ts has been increased, the problems in tests and with the style module have been fixed

Good Job. Congratulations.

**Pull request merged**